### PR TITLE
build Contact page

### PR DESCRIPTION
**Pull request by austin-pham-goldenowl**

### CODE REVIEW

File: src/pages/Contact/index.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import MainHeader from "../../components/MainHeader";
+import SubHeader from "../../components/SubHeader";
+import BreadCrumb from "../../components/BreadCrumb";
+import Footer from "../../components/Footer";
+import "./index.sass";
```
Comment (hansanwok) [inline src/pages/Contact/index.js]: move it to bottom import to seperate section

File: src/components/SubHeader/index.js [hunk 2]
```diff
@@ -78,14 +78,16 @@ class SubHeader extends Component {
...
-            alt="arrow-right"
-          />
+          <Link to="/contact-us" className="d-flex">
+            <h4 className="sub-header__contact-title sub-header__text">
+              Contact us
+            </h4>
+            <img
+              className="sub-header__arrow-right"
+              src={arrowRight}
+              alt="arrow-right"
```
Comment (hansanwok) [inline src/components/SubHeader/index.js]: alt should be "contact us"

File: src/pages/App/index.sass [hunk 3]
```diff
@@ -64,6 +64,25 @@ a
...
+  color: $darkGray
+
+input, select, textarea
+  font-family: ProximaNovaRegular
+  color: #85868f !important
+  border-radius: unset !important
+  margin-bottom: 1.25rem
+
+textarea
+  padding: 0.4375rem 0.75rem !important
```
Comment (hansanwok) [inline src/pages/App/index.sass]: why be so much !important

Comment (austin-pham-goldenowl) [reply src/pages/App/index.sass]: because I need to override bootstrap class styles

**Pull request merged**